src/build.d

```diff
@@ -78,11 +78,7 @@ ENABLE_SANITIZERS     Build dmd with sanitizer (e.g. ENABLE_SANITIZERS=address,u
 
 Targets
 -------
-
-all                   Build dmd
-unittest              Run all unittest blocks
-clean                 Remove all generated files
```

Comment:
Clean will disappear from the output now :/

Reply:
You're right, I've added clean as a dependency. Done.